sdk/identity/identity/src/credentials/managedIdentityCredential.ts

```diff
@@ -215,25 +261,34 @@ export class ManagedIdentityCredential implements TokenCredential {
     scopes: string | string[],
     options?: GetTokenOptions
   ): Promise<AccessToken | null> {
+    const span = createSpan("ManagedIdentityCredential-getToken", getSpanOptions(options));
+    span.start();
+    if (!options) {
+      options = {};
+    }
+    options.spanOptions = {
+      parent: span
+    };
```

Comment:
We have `createSpan` method where we can update the spanOptions with the parent info. We shouldn't need to do this here. What do you think about updating `createSpan()` to take options of type `GetTokenOptions` and updating the `spanOptions.parent` there?